ArkEcosystem/explorer
# refactor: optimize delegate voters count command

## Summary

https://app.clickup.com/t/1hk21np

Passed from **284.31secs** to `2.59s` 



### To test:

1. Localize some wallets with votes and open the voters page `wallets/{address}/voters` (for example https://dexplorer.ark.io/wallets/DNjuJEDQkhrJ7cA9FZ2iVXt5anYiM8Jtc9/voters)
2. see votes,
3. clear cache and run the command `php artisan explorer:cache-delegate-voter-counts`
4. Refresh voters page and expect exact same results

## Checklist
